Shell/Options.cpp

```diff
@@ -2587,7 +2587,7 @@ bool Options::RatioOptionValue::readRatio(const char* val, char separator)
       return false;
     }
     char copy[128];
```

Comment:
Safety. Already checked by preconditions but worth having as can help compiler tell when e.g. buffer is too small. Not in a hot loop.

Reply:
When you do a final pass of this, could you please factor out "128" to a "const int COPY_SIZE" or something similar?